settings/language-sqf-macros-cba.json

```diff
@@ -0,0 +1,79 @@
+{
+  ".source.sqf, .source.cpp, .source.hpp, .source.ext": {
+    "autocomplete": {
+      "symbols": {
+        "macro-cba": {
+          "typePriority": 0,
+          "suggestions": [{
+              "description": "Expands to: prefix_component",
+              "text": "ADDON",
+              "rightLabel": "CBA Macro"
+            }, {
+              "description": "Expands to: prefix",
+              "text": "PREFIX",
+              "rightLabel": "CBA Macro"
+            }, {
+              "description": "Expands to: component",
+              "text": "COMPONENT",
+              "rightLabel": "CBA Macro"
+            }, {
+              "description": "Expands to: function = compile preProcessFileLineNumbers \"functionPath\"",
+              "snippet": "PREP(${1:function})$0",
+              "rightLabel": "CBA Macro"
+            }, {
+              "description": "Expands to: prefix_component_variable",
+              "snippet": "GVAR(${1:variable})$0",
+              "rightLabel": "CBA Macro",
+              "descriptionMoreURL": "http://ace3mod.com/wiki/development/coding-guidelines.html#4-1-module-pbo-specific-macro-usage"
+            }, {
+              "description": "Expands to: \"prefix_component_variable\"",
+              "snippet": "QGVAR(${1:variable})$0",
+              "rightLabel": "CBA Macro",
+              "descriptionMoreURL": "http://ace3mod.com/wiki/development/coding-guidelines.html#4-1-module-pbo-specific-macro-usage"
+            }, {
+              "description": "Expands to: prefix_component_variable",
+              "snippet": "EGVAR(${1:component},${2:variable})$0",
+              "rightLabel": "CBA Macro",
+              "descriptionMoreURL": "http://ace3mod.com/wiki/development/coding-guidelines.html#4-1-module-pbo-specific-macro-usage"
+            }, {
+              "description": "Expands to: prefix_component_fnc_functionName",
+              "snippet": "FUNC(${1:functionName})$0",
```

Comment:
@jokoho48 You can create a new issue if you see this is a problem or we should add something.